### DISCUSSION TITLE
Database restore in v6.0.105

### DISCUSSION THREAD

**Discussion by dushko-murtovski** [ravendb/ravendb#19287] [General]
Hi,

Not sure if i should be posting this here, so please remove it if it is not relavant.

After updating to version 105 from 101 i can no longer make restore into new database. It seems that the backup file is no longer recognised as a valid backup. If i make the restore on 101 and then switch to 105 the database works just fine.
I have the Community license.

Has anybody else experinced this?

Regards.

**Comment by ayende** [comment]
That should not happen, no.

A backup from an older version should always be available to restore in a newer one.
Can you send us such a sample file?

**Reply by dushko-murtovski** [reply]
So i found the problem. 
The name of the folder where the backup is needs to be the same with the same name of the folder on the server. Basically you need to copy not the file but the folder. 
Ok, maybe it is something i overlooked and maybe there is a good reason for this but it is a little strange.

Thanks for the fast responce by the way.
